services/dune.tsx

```diff
@@ -0,0 +1,73 @@
+import {strict as assert} from 'node:assert'
+
+const DUNE_API_KEY = process.env.DUNE_API_KEY
+assert(DUNE_API_KEY, "DUNE_API_KEY environment var is required")
+
+// TODO: getFromDune will be moved in a future PR to the SDK
+interface MetadataQuery {
+  executed_at: string,
+  job_id: string,
+  query_duration_millis: number,
+  query_version: number,
+  result_bytes: number
+  result_rows: number
+}
+
+interface GetFromDuneResult<T> {
+  metadata: MetadataQuery
+  column_names: string[]
```

Comment:
Couldn't we just ignore the unused content here by not parsing it? I suppose we are at the mercy of the query owner not changing the column name, but we have assumed (below) the columns name is specific. We could also assert there is only one column and fetch the column by the name that is in this list.

Reply:
I will comment out. My thinking was that we can use this later on to do nicer generic charts, but I think you are right. Lets go there when we get there. For now, is no needed, so better to add the minimum

Reply:
Do you think this might also apply to some of the unnecessary fields in metadata?